source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
+- `https://{register-name}.register.gov.uk/`
+
+## Next version
+You can use this version of the API to test against backwards incompatable API changes before we release a new version of the API.
+
+Access it using this base URL:
+
+- `https://{register-name}.register.gov.uk/next/`
+
+**WARNING: Do not use this URL in production, as its API may change without warning.**
```

Comment:
Unfortunately we shouldn't use bold for anything other than interface text. You could use the solution Jon Heslop came up with for me on Pay to get the warning symbol out of the Design System: https://github.com/alphagov/pay-tech-docs/blob/master/source/payment_flow_overview/index.html.md.erb